source/documentation/deploying_services/index.md

```diff
@@ -6,7 +6,11 @@ Many 12-factor applications rely on backing services such as a database, an emai
 
 In Cloud Foundry, backing and routing services are referred to as 'services' and are available through the Cloud Foundry ``cf marketplace`` command.
 
-Currently, the only services available in the marketplace are the backing [PostgreSQL database service](/#using-postgresql) and [MySQL database service](/#using-mysql).
+Currently, the following database services are available in the marketplace:
+[PostgreSQL](/#using-postgresql)
+[MySQL](/#using-mysql)
+MongoDB
```

Comment:
@combor I did not delete the links to #using-postgresql and #using-mysql yet as I wanted to check if my proposed changes would be accepted first.

@bleach I know we spoke about keeping the changes unmerged until MongoDB is available; however if we accept the removal of duplicated content, then maybe we could make those changes live and keep the MongoDB changes in reserve until it is available?